docs/build/group-chat.md

```diff
@@ -696,11 +706,15 @@ stream.stop()
 
 ## Sync group chats
 
-Calling `sync()` for a group or groups gets any updates since the last sync and adds them to the local database. Be sure to periodically synchronize each group chat to ensure your app has the latest group chat details, including the most recent messages, member list, and group chat details, for example.
+Calling `sync()` for a group or groups gets any updates since the last sync and adds them to the local database.
+
+Sync group chats anytime a user accesses a conversation view. This sync is especially important if a user backgrounds (switches away from) your app, then reopens the app and accesses a conversation view. 
+
+When the user reopens the app, the app might be using an old epoch (snapshot) of data. This is even more likely to be true if there were many group membership changes during the time the app was backgrounded. A group chat sync bumps the app to the latest epoch, helping to ensure that your user always sees the current group chat, including the most recent messages, member list, and group chat details, for example.
```

Comment:
Maybe we could make it clearer what an old epoch is. An old epoch is an old decryption/encryption key for the group. Every time a member is added or removed the decryption/encryption keys get rotated so that old members cannot decrypt messages going forward and no members can not decrypt old messages from the group before they joined etc... If an app doesn't sync often the epochs for a user can get so far behind that they can no longer decrypt messages (read new messages) or encrypt messages (send messages that can be decrypted by others in the group). This is why syncing often is important.